pierce-portfolio/portfolio/src/main/webapp/style.css

```diff
@@ -0,0 +1,31 @@
+#content {
+  margin-left: auto;
+  margin-right: auto;
+  width: 650px;
+}
+
+#greeting-container {
+  margin-top: 20px;
+}
+
+/* Style the links inside the navigation bar */
+.topnav a {
+  float: left;
```

Comment:
Can you order the CSS style alphabetically.

Reply:
Okay